src/main/kotlin/no/nav/familie/ks/sak/integrasjon/tilbakekreving/TilbakekrevingKlient.kt

```diff
@@ -26,6 +30,24 @@ class TilbakekrevingKlient(
 
         return finnesBehandlingsresponsDto.finnesÅpenBehandling
     }
+
+    fun hentForhåndsvisningTilbakekrevingVarselbrev(forhåndsvisVarselbrevRequest: ForhåndsvisVarselbrevRequest): ByteArray {
+        val uri = URI.create("$familieTilbakeUri/dokument/forhandsvis-varselbrev")
+
+        return kallEksternTjeneste(
+            tjeneste = "familie-tilbake",
+            uri = uri,
+            formål = "Henter forhåndsvisning av varselbrev"
+        ) {
+            postForEntity(
+                uri = uri,
+                payload = forhåndsvisVarselbrevRequest,
+                httpHeaders = HttpHeaders().apply {
+                    accept = listOf(MediaType.APPLICATION_PDF)
+                }
+            )
+        }
+    }
 }
 
 data class FinnesBehandlingsresponsDto(val finnesÅpenBehandling: Boolean)
```

Comment:
Denne kodelinja er ikke endret i denne PR'n her men:
Kan vi rename den? Siden den eksisterer i TilbakekrevingKlient så antar jeg at den bare blir brukt til å sjekke om det finnes en tilbakekrevingsbehandling. Kanskje rename til noe ala:

```suggestion
data class FinnesTilbakekrevingsBehandlingsresponsDto(val finnesÅpenBehandling: Boolean)
```

Reply:
endret til FinnesTilbakekrevingBehandlingsresponsDto